db/migrate/20200403175026_add_fields_to_pruebas.rb

```diff
@@ -0,0 +1,5 @@
+class AddFieldsToPruebas < ActiveRecord::Migration[6.0]
+  def change
+    add_column :pruebas, :description, :string
```

Comment:
You crate your `pruebas` table with `description attribute.
You can not add a second column with same name. Drop your db, set it up again and you will see what I mena.